convert_imgs.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-for i in ../images/m*/**/*.jpg;
-do
+# This script helps to fix invalid photos shot on some phones
+# (e.g. on iPhone XR)
+
+for i in data/raw/m*/**/*.jpg; do
+  echo Converting $i
```

Comment:
```suggestion
  echo Converting "$i"
```

Reply:
I believe that for echoing trivial strings there is no difference if expanded variable is quoted, but for safety and consistency sake I added them. By the way I included another comment explaining why we glob over "m*" dirs.